Sources/Danger/GitHubDSL.swift

```diff
@@ -17,489 +14,449 @@ public struct GitHub: Decodable {
-    public let base: GitHubMergeRef
-
-    /// The state for the pull request: open, closed, locked, merged.
-    public let state: PullRequestState
-
-    /// A boolean indicating if the pull request has been locked to contributors only.
-    public let isLocked: Bool
-
-    /// A boolean indicating if the pull request has been merged.
-    public let isMerged: Bool?
-
-    /// The number of commits in the pull request.
-    public let commitCount: Int?
-
-    /// The number of comments in the pull request.
-    public let commentCount: Int?
-
-    /// The number of review-specific comments in the pull request.
-    public let reviewCommentCount: Int?
-
-    /// The number of added lines in the pull request.
-    public let additions: Int?
-
-    /// The number of deleted lines in the pull request.
-    public let deletions: Int?
-
-    /// The number of files changed in the pull request.
-    public let changedFiles: Int?
-
-    /// The milestone of the pull request
-    public let milestone: GitHubMilestone?
-
-    /// The link back to this PR as user-facing
-    public let htmlUrl: String
 }
 
-// MARK: - GitHubUser
-
-/// A GitHub user account.
-public struct GitHubUser: Decodable, Equatable {
-    // MARK: - CodingKeys
-
-    enum CodingKeys: String, CodingKey {
-        case id
-        case login
-        case userType = "type"
+extension GitHub {
+    /// A GitHub user account.
+    public struct User: Decodable, Equatable {
+        enum CodingKeys: String, CodingKey {
+            case id // swiftlint:disable:this identifier_name
+            case login
+            case userType = "type"
+        }
+
+        public enum UserType: String, Decodable {
+            case user = "User"
+            case organization = "Organization"
+            case bot = "Bot"
+        }
+
+        /// The UUID for the user organization.
+        public let id: Int // swiftlint:disable:this identifier_name
+
+        /// The handle for the user or organization.
+        public let login: String
+
+        /// The type of user: user, organization, or bot.
+        public let userType: UserType
     }
-
-    // MARK: - UserType
-
-    public enum UserType: String, Decodable {
-        case user = "User"
-        case organization = "Organization"
-        case bot = "Bot"
-    }
-
-    // MARK: - Properties
-
-    /// The UUID for the user organization.
-    public let id: Int
-
-    /// The handle for the user or organization.
-    public let login: String
-
-    /// The type of user: user, organization, or bot.
-    public let userType: UserType
-}
-
-// MARK: - GitHubTeam
-
-/// A GitHub team.
-public struct GitHubTeam: Decodable, Equatable {
-    /// The UUID for the team.
-    public let id: Int
-
-    /// The team name
-    public let name: String
 }
 
-// MARK: - GitHubRequestedReviewers
+extension GitHub {
+    /// A GitHub team.
+    public struct Team: Decodable, Equatable {
+        /// The UUID for the team.
+        public let id: Int
```

Comment:
<!--
  0 failure: 
  1 warning:  Variable name sho...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitHubDSL.swift;
  Line: 166;
-->

- :warning: Variable name should be between 3 and 40 characters long: 'id' (`identifier_name`)